src/RTG/ChatFilter/Loader.php

```diff
@@ -33,64 +33,41 @@
 use pocketmine\event\player\PlayerChatEvent;
 
 class Loader extends PluginBase implements Listener {
-
-    public $words;
+    public $cfg;
 
     public function onEnable() {
-        $this->getServer()->getPluginManager()->registerEvents($this, $this);
-        $this->words = array();
+        $this->getServer()->getPluginManager()->registerEvents($this, $this)
         $this->cfg = new Config($this->getDataFolder() . "words.txt", Config::ENUM);
-
-        $this->words = $this->cfg->getAll();
-    }
-
-    public function onSave() {
-        $this->cfg = new Config($this->getDataFolder() . "words.txt", Config::ENUM);
-        $this->cfg->setAll($this->words);
-        $this->cfg->save();
     }
 
     public function onCommand(CommandSender $sender, Command $command, $label, array $args) {
         switch (strtolower($command->getName())) {
-
```

Comment:
Sorry about these empty lines. They make the code hard to read for me. If you love them, just add them back :grin: